flume-ng-sdk/src/main/java/org/apache/flume/event/SimpleEvent.java

```diff
@@ -64,4 +64,7 @@ public String toString() {
     return "[Event headers = " + headers + ", body.length = " + bodyLen + " ]";
   }
 
+  public void notifySource() {}
+
+  public void commit() {}
```

Comment:
Can you please add javadoc to clarify usage?

Reply:
sure.